.github/workflows/ci.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches: [ main, feature/*, hotfix/* ]
   push:
     branches: [ main, feature/*, hotfix/* ]
+  workflow_dispatch:
```

Comment:
What is the purpose of this change?

Reply:
Leftover from the try to get the snapshot ci running on pull requests. Sorry about that